src/listModel.js

```diff
@@ -0,0 +1,17 @@
+export default [
+  {
+    description: 'take out trash',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'wash dishes',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'walk dog',
+    completed: false,
+    index: 2,
+  },
+];
```

Comment:
- [ ] The task object's values for `completed` is updated upon user actions. I have an error with the complete action, please fix it.

![complete](https://user-images.githubusercontent.com/55635977/144318879-88e0e5de-b3a2-4b27-9dcf-581f7f67388c.png)